### PR TITLE
Update usage.rst

### PR DESCRIPTION
**Pull request by TrexaSolutions**
Translation from English to Spanish

### CODE REVIEW

File: es/bake/usage.rst [hunk 1]
```diff
@@ -1,15 +1,103 @@
...
+    - model
+    - plugin
+    - shell
+    - shell-helper
+    - template
+    - test
+
+    By using 'cake bake [name]' you can invoke a specific bake task.
+
+Puede obtener más información sobre lo que realiza cada una de las actividades y sus opciones usando el parametro '--help' option:
```
Comment (markstory) [inline es/bake/usage.rst]: Preformatted blocks need to be preceded by `::` not `:`.

File: es/bake/usage.rst [hunk 2]
```diff
@@ -1,15 +1,103 @@
...
+    --no-test         Do not generate a test skeleton.
+    --no-actions      Do not generate basic CRUD action methods.
+
+    Arguments:
+
+    name  Name of the controller to bake. Can use Plugin.name to bake
+        controllers into plugins. (optional)
+    
+Bake Temas / Templates
+===========
```
Comment (markstory) [inline es/bake/usage.rst]: The underlines need to match the text length in rst.

File: es/bake/usage.rst [hunk 3]
```diff
@@ -1,15 +1,103 @@
...
+Ver sección :ref:'database configuration <database-configuration>' para mas información.
+
+A fin de comenzar con la ejecución del comando, debe abrir la consola de windows y ejecutar "Cake Bake"
+
+1. Ir a Inicio (Start) > Ejecutar (Run)
+2. Escribir "cmd" y presionar 'Enter'
+3. Navegar hasta llegar a la carpeta de instalación del cake
+4. Acceder a la carpeta 'bin'
+5. Escribir 'Cake bake' lo cual deberá devolver un listado con todas las tareas/actividades disponibles.
+El resultado debería ser algo similar a lo siguiente:
```
Comment (markstory) [inline es/bake/usage.rst]: This should end with `::` not `:`

**Comment by TrexaSolutions** [comment]
Hi Mark and Ravage, Thanks for the review you made on my translation. I've already fixed the issues and committed them. Should I perform any other activity or just wait for someone to review and approve /correct it? Sorry for asking but I'm not so expert in Hithub. Thank you. Regards. Ignacio

**Comment by markstory** [comment]
Thanks @TrexaSolutions. I cleaned up a bit of formatting post merge.

**Pull request merged**